tensorrt_llm/_torch/pyexecutor/llm_request.py

```diff
@@ -35,10 +35,199 @@
 }
 
 
+class LogitsStorage:
+    beam_width: int = -1
```

Comment:
IIUC, are these two members ClassVar[int]?

Reply:
They are lazily initialized fields outside `__init__`, and these are type hints for them. I updated to setup placeholder values in `__init__`.